lib/src/main/java/uk/gov/di/data/lep/library/services/Mapper.java

```diff
@@ -2,18 +2,26 @@
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
 import com.fasterxml.jackson.dataformat.xml.XmlMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 
+import java.time.temporal.TemporalAccessor;
+
 public class Mapper {
     private Mapper() {
         throw new IllegalStateException("Utility class");
     }
 
     public static ObjectMapper objectMapper() {
+        var temporalAccessorModule = new SimpleModule();
+        temporalAccessorModule.addSerializer(TemporalAccessor.class, new TemporalAccessorSerializer());
```

Comment:
I think you should be able to add these as an annotation to the fields we want to serialize/deserialize rather than as a mapper level config
```@JsonSerialize(using =...)```
which might be more resilient

Reply:
Specifically, I think
```java
public record IsoDate(
    String description,
    @JsonSerialize(using = TemporalAccessorSerializer.class)
    @JsonDeserialize(using= TemporalAccessorDeserializer.class)
    TemporalAccessor value
) {
}
```

for the IsoDate works

Reply:
Instead, am making a PartialDate serializer at top level

Reply:
Not doing this and doing what Olly said